doc/examples/README.md

```diff
@@ -0,0 +1,10 @@
+# Taipy GUI Examples
+
+This directory contains an organized set of Python script files that can
```

Comment:
Should we mention the fact that these examples are used and linked fin the doc and add a back link?

Reply:
It is indicated that these are referenced from the doc.
However a back link is impossible due to the release branch name that changes...